cli/src/config.rs

```diff
@@ -1059,6 +1059,9 @@ pub struct _Validator {
     // Warp the ledger to WARP_SLOT after starting the validator.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub warp_slot: Option<String>,
+    // Deactivate one or more features.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub deactivate_feature: Option<Vec<String>>
```

Comment:
Just a quick comment, no change is necessary: `Anchor.toml` currently uses [tables](https://toml.io/en/v1.0.0#table) instead of [arrays](https://toml.io/en/v1.0.0#array) e.g.

https://github.com/coral-xyz/anchor/blob/c66b3232938186623c535cec8deba6b98353420b/cli/src/config.rs#L990-L993

but as long as the values don't (and won't) have other keys associated with it, like in this case, I think arrays are better to use.

Reply:
Thx for quick review. 
- Updated the CHANGELOG. 
- Agree w/ you that Array makes more sense here.